search/api_test.py

```diff
@@ -82,6 +82,9 @@ def test_create_search_obj_program_limit(self):
             'bool',
             should=[
                 Q('term', **{'program.id': self.program.id})
+            ],
+            must_not=[
+                Q('term', **{'program.is_staff': True})
```

Comment:
this is not enough.
you need to add another user that is not staff and make sure that that user is in the search result and the user that is `staff is not`